### PR TITLE
Update list_upload_troubleshooting.md

### PR DESCRIPTION
**Pull request by Zaftula**
Updating this article with more complete examples and research from the Support team.

**Description of the change**:
**Reason for the change**:
**Link to original source**:

@ksigler7

### CODE REVIEW

File: source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md [hunk 1]
```diff
@@ -1,53 +1,100 @@
...
-3. The other columns can be blank.
+Make sure that your dates are properly formatted in your CSV. If you created your CSV with Excel, you may need to force the date format before exporting to CSV. The date format needs to be one of the following: `MM/DD/YYYY, MM/D/YYYY, M/D/YYYY, or M/DD/YYYY`
 
-**Custom fields in your CSV but not your account**
+{% anchor h3 %} Custom fields in your CSV but not your account {% endanchor h3 %}
 
 If you include a column in your CSV that is for a custom field not defined in your account, then the CSV will be rejected. We are currently working to add a feature that will let you set up custom fields as a step during the upload process.
 
-**I have more than one of the same email in my database**
+{% anchor h3 %} I have more than one of the same email in my database {% anchor h3 %}
```
Comment (ksigler7) [inline source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md]: Please update the end tag to {% endanchor h3 %} (this is what caused the build to fail)

File: source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md [hunk 2]
```diff
@@ -1,53 +1,100 @@
...
 weight: 0
 layout: page
-zendesk_id: 206317698
 navigation:
   show: true
 ---
 
- **My upload file is not accepted**
+{% info %}
+In the case of an error, we will often send an email to you in the case of a list upload failure. Please check the email from SendGrid that outlines the results of your CSV upload, including a downloadable error report.
```
Comment (Whatthefoxsays) [inline source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md]: In the case of a list upload failure, you may receive an email from SendGrid. Please check the email from SendGrid containing the results of your CSV upload, including a downloadable error report.

File: source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md [hunk 3]
```diff
@@ -1,53 +1,100 @@
...
+{% anchor h3 %} Text that looks like a number {% endanchor h3 %}
+
+Make sure that you don’t have text fields that look like numbers. Text that look like numbers are those that contain punctuation characters like a dash, underscore, parenthesis, or multiple dots.
+
+```
 - US Zip Code: 80202-1713
 - Phone numbers: “(555) 555-5555” or “555.555.5555”
 - Monetary Values with the currency indicator such as $3.50 or €5.73
+```
+{% anchor h3 %} CSV Headers can be blank {% anchor h3 %}
```
Comment (Whatthefoxsays) [inline source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md]: CSV Header

File: source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md [hunk 4]
```diff
@@ -1,53 +1,100 @@
...
 
 With SendGrid’s Marketing Campaigns feature, the unique identifier is the email address. So, if you upload the same email address multiple times, the custom field data associated with that email will be updated with each upload to the most recently uploaded/updated information. This feature helps prevent you from accidentally emailing the user after they have unsubscribed.
 
 What you may want to do is add logic to set a custom field, based on the reason why you have duplicate emails in your system (for example multiple product lines) and then segment your user to be in lists based on those custom fields.
 
 **‹ ** [Custom Fields](https://sendgrid.com/docs/User_Guide/Marketing_Campaigns/custom_fields.html)
 
+
+{% anchor h2 %} Specific Upload Errors {% endanchor h2 %}
+
```
Comment (Whatthefoxsays) [inline source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md]: Can you add a blurb here explaining these errors? Something like: Here is a list of potential errors you many encounter along with some easy fixes to help you successfully upload a CSV file. (this is terrible, I'm sure you can come up with something better)

File: source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md [hunk 5]
```diff
@@ -1,53 +1,100 @@
...
+{% anchor h3 %} We were unable to detect an email column in the CSV file headers. The email column must always have a header of "email" {% endanchor h3 %}
+
+- If there is a header labeled “email”, try moving it to the column A, so it is the first header in your list.
+- Your CSV _must_ include an email header.
+
+![]({{root_url}}/images/listupload_1.png)
+
+{% anchor h3 %} Some of your custom fields have not been selected {% endanchor h3 %}
+
+- If you have worked with your CSV multiple times, or have made some adjustments to the data, there can be some hidden characters in the CSV file.
```
Comment (Whatthefoxsays) [inline source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md]: Look for hidden characters in your CSV file if you have worked with the file multiple times and made some adjustments to the data.

File: source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md [hunk 6]
```diff
@@ -1,53 +1,100 @@
...
+- Please be sure to include headers at the top of each column. The “email” header is required for all CSV uploads. Header fields can be blank in your CSV, but must be defined later during the upload process.
+
+![]({{root_url}}/images/listupload_4.png)
+
+{% anchor h3 %} Float type conversion error {% endanchor h3 %}
+
+- Dashes or decimal places past the hundredth place need to be categorized as text_fields (-100, 123.123 are text fields, 100, 12.12 can be number fields).
+Number fields can include monetary values without the currency symbols. For example, 19.95 would be a number field, and $19.95 including the ($) would be a text field.
+- As you are uploading your CSV, make sure you select the correct fields for your Field type.
+
```
Comment (Whatthefoxsays) [inline source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md]: Categorize dashes or decimal places that are past the hundredth place as text_fields (-100, 123.123 are text fields, 100, 12.12 can be number fields).
 +Number fields can include monetary values without the currency symbols. For example, 19.95 would be a number field, and $19.95 including the ($) would be a text field.
 +- As you are uploading your CSV, select the correct fields for your Field type.

File: source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md [hunk 7]
```diff
@@ -1,53 +1,100 @@
...
+- Check your CSV for duplicate fields, especially fields that have matching information as your “email”, “first_name” and “last_name” fields. These are reserved field names, and cannot be edited.
+- For example, if you have a “Name” field, it may conflict with your “first_name” reserved field.
+
+![]({{root_url}}/images/listupload_3.png)
+
+{% anchor h3 %} We were unable to detect any headers in your CSV file {% endanchor h3 %}
+
+- Please be sure to include headers at the top of each column. The “email” header is required for all CSV uploads. Header fields can be blank in your CSV, but must be defined later during the upload process.
+
```
Comment (Whatthefoxsays) [inline source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md]: Include headers at the top of each column. The “email” header is required for all CSV uploads. Header fields can be blank in your CSV, but must be defined later during the upload process.

File: source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md (hunk 6 above)
Comment (Whatthefoxsays) [inline source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md]: Aw crap, is the first part of this an error message? or is that what's in the h3 tag?

File: source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md [hunk 8]
```diff
@@ -1,53 +1,100 @@
...
-**Missing Header**
+![]({{root_url}}/images/listupload_5.png)
 
-Your CSV should always have a header row. The rules for this are:
+{% anchor h3 %} Date formatting {% endanchor h3 %}
 
-1. You must have an “email” column.
-2. Capitalization doesn’t matter.
-3. The other columns can be blank.
+Make sure that your dates are properly formatted in your CSV. If you created your CSV with Excel, you may need to force the date format before exporting to CSV. The date format needs to be one of the following: `MM/DD/YYYY, MM/D/YYYY, M/D/YYYY, or M/DD/YYYY`
```
Comment (Whatthefoxsays) [inline source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md]: Properly format any dates in your CSV. If you created your CSV with Excel and it is not displaying properly, force the date format before exporting to CSV. Use one of the following formats: `MM/DD/YYYY, MM/D/YYYY, M/D/YYYY, or M/DD/YYYY`

**Pull request merged**